googleMapsApiModule.py

```diff
@@ -56,3 +56,10 @@ def geocoding(search_location):
     url = "https://www.google.com/maps/search/?api=1&query="
     result_url = url+address
     webbrowser.open_new(result_url)
+
+def mapsstatic(search_location):
+    address = search_location.lower()
+    address = address.replace(" ","+")
+    result_url = "https://maps.googleapis.com/maps/api/staticmap?center="+address+"&zoom=13&scale=1&size=600x350&maptype=roadmap&key="+config.key+"&format=png&visual_refresh=true&markers=size:mid%7Ccolor:0xff0000%7Clabel:L%7C"+address
```

Comment:
This is just a shared-nitpicking, that we suggested in another PR too. Please have a look at these comments and make the relevant changes to `result_url` declaration:

1. https://github.com/vishakha-lall/MapBot/pull/42#discussion_r386859182
2. https://github.com/vishakha-lall/MapBot/pull/42#discussion_r386864542

Reply:
@adityabisoi - Looks like this PR has been already merged, can you open a new PR for this tiny nitpicking?

Reply:
Ah yes, I wanted to take this up as another 'beginner' level issue to improve the string generations in all these functions, will create an issue for the same.